base/docs/Docs.jl

```diff
@@ -675,4 +675,15 @@ function hasdoc(binding::Docs.Binding, sig::Type = Union{})
 end
 
 
+"""
+    check_documented(mod::Module; all=false)
+
+Check all names in the module are documented. `all` determines which names are checked,
+following the behavior of `names(mod; all)`.
```

Comment:
Should have a cross-reference to names docs. Should probably just repeat what all means here